packages/vscode-extension/src/commands/ImportObjects.ts

```diff
@@ -28,10 +28,9 @@ export default class ImportObjects extends BaseAction {
 		}
 		const projectInfoService = new ProjectInfoService(this.getProjectFolderPath());
 
-		const relativeDestinationFolder = this.getDestinationFolder(this.activeFile);
-		if (!relativeDestinationFolder.startsWith(ApplicationConstants.FOLDERS.OBJECTS)) {
-			this.messageService.showErrorMessage(this.translationService.getMessage(IMPORT_OBJECTS.ERROR.INCORRECT_FOLDER));
-			return;
+		let relativeDestinationFolder = ApplicationConstants.FOLDERS.OBJECTS;
+		if (this.isSelectedFromContextMenu) {
+			relativeDestinationFolder = this.getDestinationFolder(this.activeFile);
```

Comment:
Shouldn't we ask about the destination folder to the user as we do in node-cli when the `/Objects` folder or the selected one from the context menu contains subfolders?

Reply:
Let´s share this point with the team, and maybe add it as an enhancement.